custom/opm/health_status.py

```diff
@@ -320,3 +356,40 @@ def adult_scale(self):
     @property
     def child_scale(self):
         return self.service_available('vhnd_child_scale_available')
+
+    @property
+    def ifa_stock_available(self):
```

Comment:
I just tried this out locally and I'm also getting all zeros, even though I do have data.  I briefly looked in to figuring out why this could be, but didn't really get anywhere with it.  I think the way you've implemented it here is correct, and there's somewhere higher up the stack that's not doing things right, either the vhnd services aren't being correctly put in the fluff table, or we're pulling them out incorrectly.  Let me know if you'd like my help in figuring it out.

BTW, this is the command I run to reindex the VhndFluff:
`$ ./manage.py ptop_fast_reindex_fluff opm custom.opm.models.VhndAvailabilityFluffPillow --bulk `